src/Network/Ethereum/Web3/Contract.purs

```diff
@@ -42,20 +44,21 @@ event ::
   EventHandler Web3 e ->
   Web3 (Either (FilterStreamState e) ChangeReceipt)
 event filter handler = do
-  eRes <- event' { ev: filter } { ev: handler } { windowSize: 0, trailBy: 0 }
-  pure $ lmap f eRes
-  where
-  f :: MultiFilterStreamState ( ev :: Filter e ) -> FilterStreamState e
-  f (MultiFilterStreamState { currentBlock, windowSize, trailBy, filters }) =
-    let
-      { ev: filter } = filters
-    in
-      { currentBlock
-      , windowSize
-      , trailBy
-      , initialFilter: filter
-      }
+  unsafeCrashWith "implement event" 0
 
+-- eRes <- event' { ev: filter } { ev: handler } { windowSize: 0, trailBy: 0 }
```

Comment:
Note that this doesn't work. Similarly, in the test-suite, we get instance errors for using `event'` which leads me to think that if we can resolve `event'`, we can restore `event`.